src/views/Statistics.vue

```diff
@@ -1,46 +1,67 @@
 <template>
   <b-card class="maincard">
     <b-card-header>
-      Cluster Statistics
+      <v-row>
+        <v-col cols="11"> Cluster Statistics </v-col>
+        <v-col cols="1">
+          <v-btn icon
+          color="primary"
+          @click="request_cluster_infos">
+            <v-icon>mdi-cached</v-icon>
+          </v-btn>
+        </v-col>
+      </v-row>
     </b-card-header>
-    <div>
-    <v-simple-table fixed-header>
-      <template v-slot:default>
-        <thead>
-        <tr>
-          <th class="first text-left">Name</th>
-          <th class="second text-left">Value</th>
-        </tr>
-        </thead>
-        <tbody>
-        <tr v-for="item in all_statistics" :key="item.index">
-          <td>{{ Object.keys(item)[0] }}</td>
-          <td>{{ Object.values(item)[0] }}</td>
-        </tr>
-        </tbody>
-      </template>
-    </v-simple-table>
+    <RequestSpinner v-if="!is_loaded" />
```

Comment:
i tested the ui and I guess this doesn't work. or I would request a change in UI because there is always a static "reload-spinner" in the upper right corner that can be used to reload the table. I don't think that this is necessary, especially because the data reload takes sooo long. I've noticed the spinner that is shown during loading the datatable, which is nice, in case someone calls this page very fast.

Reply:
I added the button, because in our older version, with each reload one could get the newest data.
I thought the issue was, if someone wanted the most recent data, they would have to sign out and log back in.
But I guess its not necesarry.